src/web/theme/pages/TryIt/Reinsurance/Reinsurance.scss

```diff
@@ -0,0 +1,55 @@
+.reinsurance__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: $tryItPadding;
+  & > * {
+    margin-bottom: 2em;
+  }
+
+  &__form {
+    margin: 0 auto;
+
+    input {
+      display: none;
+      ~ label {
+        &:before {
+          transition: 0.5s;
```

Comment:
Is this actually working ? I didn't know you could specify only a duration :confused:

Reply:
Yes ! this is working :)